### PR TITLE
SBX - ed api key

### PR DESCRIPTION
**Pull request by peppasnet**

### CODE REVIEW

File: ionos_sbx/marketplace/iam/apisix/templates/eurodyn-apikey-secret-sealed-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (bialask-daiteap) [inline ionos_sbx/marketplace/iam/apisix/templates/eurodyn-apikey-secret-sealed-secret.yaml:1]: Please squash the commits and prefix commit name with environment name 'sbx'

**Review by bialask-daiteap — CHANGES_REQUESTED** [review]

File: ionos_sbx/marketplace/iam/apisix/templates/eurodyn-apikey-secret-sealed-secret.yaml (hunk 1 above)
Comment (peppasnet) [reply ionos_sbx/marketplace/iam/apisix/templates/eurodyn-apikey-secret-sealed-secret.yaml:1]: Hi, I did a rebase, but there is no squash option, do i have the rights? If not, please give me access. Thanks.

Comment (bialask-daiteap) [reply ionos_sbx/marketplace/iam/apisix/templates/eurodyn-apikey-secret-sealed-secret.yaml:1]: Squashing commits is part of rebase process, it doesn't require extra rights I think.

Comment (peppasnet) [reply ionos_sbx/marketplace/iam/apisix/templates/eurodyn-apikey-secret-sealed-secret.yaml:1]: ok, I did the rebase so I think it's alright now. Thanks.

**Comment by bialask-daiteap** [comment]
Please squash commits and prefix the commit comment with env name

**Comment by peppasnet** [comment]
> Please squash commits and prefix the commit comment with env name

please delete this PR and use this one "https://github.com/DOME-Marketplace/dome-gitops/pull/495"

**Pull request closed without merging**